persistence/model/src/main/java/gov/va/vro/persistence/model/BaseEntity.java

```diff
@@ -23,6 +24,7 @@ public abstract class BaseEntity implements Persistable<UUID> {
   @Id
   @GeneratedValue(generator = "uuid")
   @GenericGenerator(name = "uuid", strategy = "uuid2")
+  @Column(columnDefinition = "BINARY(16)")
```

Comment:
To address [this test failure](https://github.com/department-of-veterans-affairs/abd-vro/runs/8216764214?check_suite_focus=true#step:8:523):

```
Caused by: org.h2.jdbc.JdbcSQLDataException: Value too long for column "ID BINARY": "CAST(X'bc4af8a8c8a6463dbe106c3b25994049' AS BINARY(16)) (16)"; SQL statement:
insert into veteran (created_at, updated_at, icn, participant_id, id) values (?, ?, ?, ?, ?) [22001-214]

```

Reply:
@dimitri-amida FYI